src/client/app/service/utils/utils.ts

```diff
@@ -0,0 +1,9 @@
+import {Injectable} from '@angular/core';
+
+@Injectable()
```

Comment:
Also, since this isn't a service, maybe we should introduce some new folder for static functions?